docs/source/features.rst

```diff
@@ -20,6 +49,25 @@ Limitations
 Dependency detection
 --------------------
 Whilst fab can automatically determine dependencies from module use statements,
-and from standalone call statements, it doesn't currently detect a dependency from a call statement
-on a single-line if statement. We can pass the analyser any dependencies which Fab can't detect,
-and they'll make their way through to the link stage.
\ No newline at end of file
+and from standalone call statements, it doesn't currently detect a dependency from a call statement on a
+single-line if statement: `IF (x .GT. 0) CALL foo()`.
```

Comment:
I presume there's a ticket for fixing this. If it doesn't already have one add a reminder to change this documentation when it is resolved.

Reply:
There wasn't one, [made it](https://github.com/metomi/fab/projects/8#card-86192673).